src/Explorer/Tabs/QueryTab.html

```diff
@@ -11,6 +11,17 @@
       Start by writing a Mongo query, for example: <strong>{'id':'foo'}</strong> or <strong>{ }</strong> to get all the
       documents.
     </div>
+    <div class="warningErrorContainer" aria-live="assertive" data-bind="visible: maybeSubQuery">
+      <div class="warningErrorContent">
+        <span><img class="paneErrorIcon" src="/Info.svg" alt="Error"/></span>

```

Comment:
I think info_color.svg is better for this case:

![image](https://user-images.githubusercontent.com/56978073/104074472-16ed7500-51c5-11eb-8115-434951b70439.png)

Reply:
Agreed

Reply:
``` suggestion
        <span><img class="paneErrorIcon" src="/info_color.svg" alt="Error"/></span>
```